plugins/armorblox/icon_armorblox/triggers/get_incidents/schema.py

```diff
@@ -0,0 +1,329 @@
+# GENERATED BY KOMAND SDK - DO NOT EDIT
+import insightconnect_plugin_runtime
+import json
+
+
+class Component:
+    DESCRIPTION = "Get a list of incidents identified by Armorblox, by default it starts polling all the incidents since last day"
+
+
+class Input:
+    
+    INTERVAL = "interval"
+    
+
+class Output:
+    
+    INCIDENTS = "incidents"
+    
+
+class GetIncidentsInput(insightconnect_plugin_runtime.Input):
+    schema = json.loads("""
+   {
+  "type": "object",
+  "title": "Variables",
+  "properties": {
+    "interval": {
+      "type": "integer",
+      "title": "Fetch_Interval",
+      "description": "Polling inteval in seconds",
+      "default": 600,
+      "order": 1
+    }
+  }
+}
+    """)
+
+    def __init__(self):
+        super(self.__class__, self).__init__(self.schema)
+
+
+class GetIncidentsOutput(insightconnect_plugin_runtime.Output):
+    schema = json.loads("""
+   {
+  "type": "object",
+  "title": "Variables",
+  "properties": {
+    "incidents": {
+      "type": "array",
+      "title": "Incidents",
+      "description": "A list of 100 incidents identified by Armorblox",
+      "items": {
+        "$ref": "#/definitions/incident"
+      },
+      "order": 1
+    }
+  },
+  "required": [
+    "incidents"
+  ],
+  "definitions": {
+    "engagement": {
+      "type": "object",
+      "title": "engagement",
+      "properties": {
+        "fwd_mail_count": {
+          "type": "string",
+          "title": "Mail Count",
+          "description": "Mail Count",
+          "order": 1
+        },
+        "reply_mail_count": {
+          "type": "string",
+          "title": "Mail Count",
```

Comment:
What are these 2 mail counts and why do they have the same title?

Reply:
@urajat , These 2 mail counts are from the incidents response in engagements section: 
([{'priority': 'HIGH', 'tagged': False, 'date': '2022-08-10T06:32:43Z', 'users': [{'name': 'Admin', 'email': 'admin@armorblox.dev', 'is_vip': False}], 'policy_names': ['Payroll Fraud'], 'title': 'payroll fraud 10 Aug 12:2:40', 'remediation_actions': ['WILL_AUTO_REMEDIATE'], 'resolution_state': 'OPEN_INCIDENT_RESOLUTION_STATE', 'object_type': 'CONTENT_MAIL', 'id': '59642', 'research_status': 'TRUE_POSITIVE', 'app_name': 'MICROSOFT_OUTLOOK', 'external_users': [{'name': 'Todd Adams', 'email': 'abigail_dietrich@fake-blair.biz', 'is_vip': False}], 'external_senders': ['abigail_dietrich@fake-blair.biz'], 'folder_categories': ['SPAM'], 'scl_score': 5, 'final_detection_tags': [{'detection_tag_id': '5', 'detection_tag_name': 'payroll'}], **_'engagements': {'fwd_mail_count': '0', 'reply_mail_count': '0'}}_**,  

> Since the value is '0' here, we have added title as Mail count for both the mails.

Reply:
Please use Forwarded Mail Count and Reply Mail Count respectively

Reply:
Okay.